test/data/search/8bins20window0error50pattern1overlap.out

```diff
@@ -0,0 +1,32 @@
+0	0,
+1	0,
+2	0,
+3	0,
+4	1,
+5	1,
+6	1,
+7	1,
+8	2,
+9	2,
+10	2,
+11	2,
+12	3,
+13	3,
+14	3,
+15	3,
+16	4,
+17	4,
+18	4,
+19	4,
+20	5,
+21	5,
+22	5,
+23	5,
+24	6,
+25	6,
+26	6,
+27	6,
+28	7,
+29	7,
+30	7,
+31	7,
```

Comment:
This is the actual ground truth: 4 queries were sampled from each of the bins. But the search results are sensitive to the input parameters {errors, pattern length, overlap}.